### PR TITLE
Search Form A11y Adjustments

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Adds some minor syntax changes to improve screen reader behavior. Relies on https://github.com/opentripplanner/otp-ui/pull/520

### CODE REVIEW

File: lib/components/app/batch-routing-panel.js
Comment (binh-dam-ibigroup) [inline lib/components/app/batch-routing-panel.js:41]: Convert to class method.

**Comment by binh-dam-ibigroup** [comment]
Typing in the `LocationField` crashes the UI for me...

**Comment by binh-dam-ibigroup** [comment]
> Typing in the `LocationField` crashes the UI for me...

Disregard, it worked again after clearing `node_modules`.

File: lib/components/form/connect-location-field.js
Comment (binh-dam-ibigroup) [inline lib/components/form/connect-location-field.js:86]: Won't hold this agains this PR, but the invalid state of the underlying `<input>` is not "seen" in ARIA dev tools.

**Review by binh-dam-ibigroup — APPROVED** [review]

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks good to me!!

**Comment by miles-grant-ibigroup** [comment]
Will merge after fixing the calltaker html

File: lib/components/form/connect-location-field.js
Comment (binh-dam-ibigroup) [reply lib/components/form/connect-location-field.js:86]: This might actually be an issue with ARIA dev tools that does not render the invalid flags on non-standard elements.

**Pull request merged**
